### ISSUE TITLE
Could not find inferPlatformFromProbeIDs function

### ISSUE THREAD

**Issue by jtxyz16** [zwdzwd/sesame#65]
betas =  openSesame(sdfs, platform = "MM285", prep = "TQCDPB", manifest = mft)
Error: BiocParallebetas =  openSesame(sdfs, platform = "MM285", prep = "TQCDPB", manifest = mft)
Error: BiocParallel errors
  1 remote errors, element index: 1
  14 unevaluated and other errors
  first remote error: could not find function "inferPlatformFromProbeIDs"l errors
  1 remote errors, element index: 1
  14 unevaluated and other errors
  first remote error: could not find function "inferPlatformFromProbeIDs"

**Comment by zwdzwd** [comment]
inferPlatformFromProbeIDs is in the updated sesameData package which sesame depends on. Please update and see if problem persist

**Comment by jtxyz16** [comment]
Thank you for the speedy reply. It's fixed now. I have another issue. I want to use SigDFsToRGSet, for downstream usage in compartmap (https://rdrr.io/bioc/compartmap) . But I'm having this error. Can you help me convert the SeSame file into an rgSet? I'm processing mouse data.

Error in platformSmToMinfi(pt) : 
  plf %in% c("IlluminaHumanMethylationEPIC", "IlluminaHumanMethylation450k",  .... is not TRUE

**Comment by zwdzwd** [comment]
Yeah, I am not sure the mouse manifest for minfi is around yet. but I can add that if there is one.
